codescan/schema.go

```diff
@@ -216,7 +216,25 @@ func (s *schemaBuilder) buildFromDecl(_ *entityDecl, schema *spec.Schema) error
 				if ti.IsBuiltin() {
 					break
 				}
+
 				if ti.IsType() {
+					if typeWithTypeParams, ok := ti.Type.(interface{ TypeParams() *types.TypeParamList }); ok && typeWithTypeParams.TypeParams() != nil && typeWithTypeParams.TypeParams().Len() > 0 {
```

Comment:
would be nice to throw a proper error for types wtihout type args (only type params) as we know, they don't have types specified. But I'm afraid we'll get to a lot of interface conversions which will get dirty really quick

Current approach will yield the same error as before: 
```
panic: WARNING: can't determine refined type T (*types.TypeParam)
```